### PR TITLE
Create a LibreCores CI Image for OpenRISC, RISC V Image now inherits the main LibreCores CI one 

### PR DESCRIPTION
**Pull request by Nancy-Chauhan**
Solves #19 #18

### CODE REVIEW

File: librecores-ci-riscv/Dockerfile [hunk 1]
```diff
@@ -20,73 +20,17 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
-FROM ubuntu:16.04
+FROM librecores/librecores-ci:latest
```
Comment (oleg-nenashev) [inline librecores-ci-riscv/Dockerfile]: pin the release version maybe? We will suffer from race conditions if we keep latest here

File: librecores-ci-openrisc/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+RUN curl --remote-name --location \
+  https://github.com/stffrdhrn/gcc/releases/download/or1k-9.0.0-20181113/or1k-elf-9.0.0-20181112.tar.xz 
+RUN  tar xC $HOME/tools -f $HOME/src/tools/or1k-elf-9.0.0-20181112.tar.xz
+
+ENV PATH="$HOME/tools/or1k-elf/bin:${PATH}"
+
+# Download and compile or1k-tests
+RUN cd $HOME/src/tools && \
+git clone https://github.com/openrisc/or1k-tests.git ; &&\
```
Comment (oleg-nenashev) [inline librecores-ci-openrisc/Dockerfile]: It picks latest from the repository and makes the build not reproducible. I suggest pinning a commit ID or a tag

File: librecores-ci-openrisc/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#Get toolchain
+RUN mkdir -p $HOME/src/tools \
+    mkdir -p $HOME/tools \
+    cd $HOME/src/tools \
+    export TRAVIS_BUILD_DIR=/src
+
+RUN apt-get update && apt-get install -y curl 
+
+RUN curl --remote-name --location \
+  https://github.com/stffrdhrn/gcc/releases/download/or1k-9.0.0-20181113/or1k-elf-9.0.0-20181112.tar.xz
```
Comment (oleg-nenashev) [inline librecores-ci-openrisc/Dockerfile]: Move version to a constant / `ARG` ?

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
With the current version the build will not be reproducible. I suggest pinning all versions instead of relying on `latest`

**Review by oleg-nenashev — APPROVED** [review]
Fine with me. Is it still a draft? Or should I merge it and see whether it works on DockerHib?

File: librecores-ci-openrisc/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+LABEL Description="This is the default LibreCores CI Image for openrisc builds" Vendor="Librecores" Version="2019.1"
+
+WORKDIR /tmp/src/tools
+
+RUN export TRAVIS_BUILD_DIR=/src 
+RUN export HOME=/tmp
+
+#Get toolchain
+    
+RUN apt-get update && apt-get install -y curl
```
Comment (oleg-nenashev) [inline librecores-ci-openrisc/Dockerfile]: This part can be optimized to reduce number and size of layers, see LCCI

**Comment by Nancy-Chauhan** [comment]
@oleg-nenashev This should work now.

**Comment by stffrdhrn** [comment]
This looks nice to have all the tools installed

**Review by oleg-nenashev — APPROVED** [review]
Would be nice to have some documentation, but we can do it later

**Comment by oleg-nenashev** [comment]
The PR was amended in #25

**Pull request merged**